client/settings/payments-and-transactions-section/data-mock.js

```diff
@@ -11,7 +11,7 @@ const makeToggleHook = ( initialValue = false ) => () => {
 	return [ value, toggleValue ];
 };
 
-export const useManualCapture = makeToggleHook( false );
+export const useManualCapture = makeToggleHook( true );
```

Comment:
Waiting on https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1812

When changing `true` to `false`, the checkbox won't appear.


With "automatic" capture (flag switched to `false`) in the settings (no checkbox is displayed, user can continue to the next step):
![2021-10-04 15 40 40](https://user-images.githubusercontent.com/273592/135921868-c57233ce-764c-4624-a44b-1724bf850469.gif)



With "manual" capture (flag switched to `true`) in the settings:
![2021-10-04 15 28 44](https://user-images.githubusercontent.com/273592/135920237-34ea7418-3896-4b87-ac64-e999f082dc28.gif)